src/app/window-actions.ts

```diff
@@ -85,31 +85,17 @@ const saveWindowSettings = async (): Promise<void> => {
   }
 };
 
-const windowMaximized = async (): Promise<void> => {
```

Comment:
as per my understanding this logic is already existing in `saveWindowSettings` function @KiranNiranjan

Reply:
Yes @sbenmoussati